portfolio.css

```diff
@@ -0,0 +1,578 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	margin:0 auto;
+	font-family: 'Major Mono Display';
+	background-color: #C8C8C8;
+}
+
+header {
+	color: #ffffff;
+	background-color: #EF1B1B;
+	opacity: 80%;
+	display: flex;
+	justify-content: space-between;
+	padding: 20px;
+}
+
+.name {
+	font-size: 70px;
+	padding: 15px;
+	border: white solid 1px;
+}
+
+.links {
+	margin-top: 30px;
+}
+
+.links a {
+	font-size: 40px;
+	color: #ffffff;
+	text-decoration: none;
+	padding: 20px 20px 20px 20px;	
+	border: 30px 0 0 0;
+}
+
+.gray__box {
+	background-color: #C8C8C8;
+	height: 90px;
+	width: 60%;
+	position: absolute;
+	top: 122px;
+	right: 0%;
+}
+
+/* piping party begins */
+
+.about__pipe1 {
+	background-color: #EF1B1B;
+	height: 90px;
+	width: 8px;
+	position: absolute;
+	top: 120px;
+	right: 50%;
+}
+
+.port__pipe1 {
+	background-color: #EF1B1B;
+	height: 60px;
+	width: 8px;
+	position: absolute;
+	top: 120px;
+	right: 28%;
+}
+
+.contact__pipe1 {
+	background-color: #EF1B1B;
+	height: 30px;
+	width: 8px;
+	position: absolute;
+	top: 120px;
+	right: 10%;
+}
+
+.about__pipe2 {
+	background-color: #EF1B1B;
+	height: 8px;
+	width: 525px;
+	position: absolute;
+	/*top: 200px;
+	right: 19%; */
+	top: 82px;
+	right: -6500%;
+}
+/* 
+.port__pipe2 {
+	background-color: #EF1B1B;
+	height: 60px;
+	width: 8px;
+	position: absolute;
+	top: 202px;
+	right: 28%;
+} */
+
+.contact__pipe2 {
+	background-color: #EF1B1B;
+	height: 30px;
+	width: 8px;
+	position: absolute;
+	top: 120px;
+	right: 10%;
+}
+
+
+
+/* piping party ends */
+
+.intro {
+	color: #ffffff;
+	background-color: #EF1B1B;
+	opacity: 80%;
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	padding: 30px;
+	margin: 20px 300px 20px 20px;
+    object-fit: cover;
+    border-radius: 1%;
+}
+
+.intro h1 {
+	font-size: 60px;
+}
+
+.intro p {
+	font-size: 40px;
+}
+
+.profile__picture {
+	border: solid;
+    object-fit: cover;
+    border-radius: 100%;
+	margin: 20px;
+	width: 90%;
+	object-fit: scale-down;
+}
+
+
+.about {
+	color: #ffffff;
+	background-color: #EF1B1B;
+	text-align: center;
+	list-style-position: inside;
+	opacity: 80%;
+	padding: 50px 30px 20px 30px;
+	margin: 20px 200px 20px 20px;
```

Comment:
you're having to adjust the right margin a lot to maintain scale when resizing the window, would a percentage based solution be better?